kubernetes/samples/scripts/common/utility.sh

```diff
@@ -926,3 +926,27 @@ function checkService(){
  done
  echo "Service [$svc] found"
 }
+
+# Get pod name when pod available in a given namespace
+function getPodName(){
+
+ max=120
+ count=1
+
+ pod=$1
+ ns=$2
+
+ pname=`kubectl get po -n ${ns} | grep -w ${pod} | awk '{print $1}'`
+ while [ -z ${pname} -a $count -le $max ] ; do
+   sleep 1
+   count=`expr $count + 1`
+   pname=`kubectl get po -n ${ns} | grep -w ${pod} | awk '{print $1}'`
+ done
+
```

Comment:
A slightly more elegant timeout pattern would be: 
max=$((SECONDS + 120))
...
while [ -z ${pname} -a $SECONDS -le $max ] ; do
...
echo "[ERROR] Could not find Pod [$pod] after $max seconds";

(The shell automatically populates SECONDS with the number of seconds since the script started.)

Reply:
Or even:
```
 pname=""
 while [ $count -le $max ] ; do
   pname=`kubectl get po -n ${ns} | grep -w ${pod} | awk '{print $1}'`
   [ -z "${pname}" ] || break
   sleep 1
 done
```

Reply:
Looks good.